### PR TITLE
Add docs for using @babel/preset-env with Webpack

### PR DESCRIPTION
**Pull request by maxloh**
Related: https://github.com/babel/babel/issues/12068, https://github.com/zloirock/core-js/issues/743#issuecomment-572074215, https://stackoverflow.com/a/59647913/11077662

### CODE REVIEW

**Comment by existentialism** [comment]
Thanks for the PR!

I feel like these caveats are not unique to `@babel/preset-env`, and belong in `babel-loader`'s docs?

**Comment by maxloh** [comment]
@exiatentialism `core-js` is a dependency of `@babel/preset-env`, so IMO we should put this to `@babel/preset-env` instead of `babel-loader`.

**Comment by existentialism** [comment]
Sure, but my argument is that _anyone_ who uses `core-js` with `webpack` needs to know this... and not everyone uses `core-js` with `@babel/preset-env`.

I'd also be okay if we augment `babel-loader`'s docs, and point a link to it from env's docs.

**Comment by maxloh** [comment]
> Sure, but my argument is that _anyone_ who uses `core-js` with `babel-loader` needs to know this... and not everyone uses `core-js` with `@babel/preset-env`.
> 
> I'd also be okay if we augment `babel-loader`'s docs, and point a link to it from env's docs.

OK 

> Sure, but my argument is that _anyone_ who uses `core-js` with `babel-loader` needs to know this... and not everyone uses `core-js` with `@babel/preset-env`.
> 
> I'd also be okay if we augment `babel-loader`'s docs, and point a link to it from env's docs.

OK. I have opened a pull request in babel-loader's repo (https://github.com/babel/babel-loader/pull/869).

We can add the link once that pull request was merged.

**Comment by maxloh** [comment]
The link isn't working now, but it should work after a new version of `babel-loader` is published

File: docs/preset-env.md
Comment (NMinhNguyen) [inline docs/preset-env.md:76]:
```suggestion
## Integrating with webpack
```

webpack is meant to be lowercase: https://webpack.js.org/contribute/writers-guide/#typesetting

**Comment by maxloh** [comment]
Can this pull request be merged? @existentialism @JLHwung @NMinhNguyen

**Review by existentialism — APPROVED** [review]

**Comment by maxloh** [comment]
@JLHwung @NMinhNguyen @hzoo @xtuc Can you help review the pull request?

**Comment by JLHwung** [comment]
> core-js is a dependency of @babel/preset-env, so IMO we should put this to @babel/preset-env instead of babel-loader.

`@babel/preset-env` does not depend on `core-js`. Instead it depends on `core-js-compat`, it is a compat table utilities for `core-js`. `@babel/preset-env` uses it to check whether user provided `include` and `exclude` options are defined in `core-js`.

Based on that, I don't think it should be placed in `@babel/preset-env` docs. I think we can extend https://babel.dev/docs/en/caveats and put that polyfills are generally untranspilable.

**Comment by maxloh** [comment]
> > core-js is a dependency of @babel/preset-env, so IMO we should put this to @babel/preset-env instead of babel-loader.
> 
> `@babel/preset-env` does not depend on `core-js`. Instead it depends on `core-js-compat`, it is a compat table utilities for `core-js`. `@babel/preset-env` uses it to check whether user provided `include` and `exclude` options are defined in `core-js`.
> 
> Based on that, I don't think it should be placed in `@babel/preset-env` docs. I think we can extend https://babel.dev/docs/en/caveats and put that polyfills are generally untranspilable.

This problem may be a common one faced by newcomers (at least for me). 

The paragraph/solution will be more easy to find if we place it in `@babel/preset-env` docs.

**Comment by maxloh** [comment]
@existentialism @JLHwung @NMinhNguyen It has been a month since the pull request was created. Can you guys discuss and make a final decision, either merge or close this pull request?

**Comment by maxloh** [comment]
Closing as there is no response from maintainers

**Pull request closed without merging**
